src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -215,9 +214,11 @@
                     }
                     if (localSource.configured) {
                         localSource.load($(this).val());
+                        return;
                     }
                     if (service.configured && service.minSize === 0) {
                         service.load($(this).val());
+                        return;
                     }
                     completions.show();
```

Comment:
is this even called anymore? i either have a localSource or a service ?

Reply:
yes, sometimes because of `&& service.minSize === 0`